source/includes/common/bucket-replication.rst

```diff
@@ -0,0 +1,167 @@
+.. start-create-replication-remote-targets-cli
+
+Use the :mc-cmd:`mc admin bucket remote add` command to create a replication target from each deployment to the appropriate bucket on the destination deployment. 
+MinIO supports *one* remote target for an origin bucket to destination bucket combination. 
+
+.. code-block:: shell
+   :class: copyable
+
+   mc admin bucket remote add ALIAS/BUCKET         \
+      https://RemoteUser:Password@HOSTNAME/BUCKETDESTINATION  \
+      --service "replication"
+
+- Replace ``ALIAS`` with the :ref:`alias <alias>` of the MinIO deployment that acts as the origin for the replication.
+- Replace ``BUCKET`` with the name of the bucket to replicate from on the origin deployment.
+- Replacete ``RemoteUser`` with the user name that has the :ref:`necessary replication permissions <minio-bucket-replication-serverside-twoway-permissions>`
+- Replace ``Password`` with the secret key for the ``RemoteUser``.
+- Replace ``HOSTNAME`` with the URL of the destination deployment.
+- Replace ``BUCKETDESTINATION`` with the name of the bucket to replicate to on the destination deployment.
+
+The command returns an :abbr:`ARN <Amazon Resource Name>` similar to the following:
+
+.. code-block:: shell
+
+   Role ARN = 'arn:minio:replication::<UUID>:BUCKET'
+
+Copy the ARN to use in the next step, noting the MinIO deployment.
+
+.. end-create-replication-remote-targets-cli
+
+.. start-create-bucket-replication-rule-console
+
+#. Select the :guilabel:`Manage` button for the bucket to replicate
```

Comment:
as much as I dislike screenshots from a maintenance point of view, it would be really nice to have here.

Otherwise you need to provide guidance for accessing and logging into the Console.

Reply:
I've reused some existing screenshots. That do not have annotations to point to bucket specific features indicated by the text, but should at least provide a visual to refer to.
I also added a first step for logging in to the Console.